plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -345,6 +348,69 @@ public void testPredicatePushdown()
         }
     }
 
+    @Test
+    public void testStringPushdownWithCollate()
+    {
+        Session session = Session.builder(getSession())
+                .setCatalogSessionProperty("sqlserver", "enable_string_pushdown_with_collate", "true")
```

Comment:
We have to run all tests with both values of this config property as column types could differ depending of the value.

Reply:
Type mapping and base connector tests are run now for both values

Reply:
Looks like it's will not be necessary anymore, after https://github.com/trinodb/trino/pull/15714#discussion_r1093614991